docs/DeveloperGuide.md

```diff
@@ -158,7 +158,16 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
-### \[Proposed\] Undo/redo feature
+### Add a new student
+On execution, the new created `Student` is then be checked if it is a duplicate of any `Student` in the current `AddressBook`, i.e. it has the exactly same `Name` **and** `Phone` with any of the current `Student`s. A `CommandException` will be thrown if it is a duplicate.
```

Comment:
Could consider using this as a template:
![image](https://github.com/user-attachments/assets/1118222b-3702-4c99-8b36-a111a678babc)

Reply:
Thank you so much for your suggestion. I have done fixing this!